src/useCookieValue/useCookieValue.ts

```diff
@@ -56,9 +56,9 @@ export type UseCookieValueOptions<
   (InitializeWithValue extends undefined
     ? {
         /**
-         * Whether to initialize state with cookie value or initialize with `undefined` state.
+         * Whether to initialize state with the cookie value or `undefined`.
          *
-         * Default to false during SSR
+         * _We suggest setting this to `false` during SSR._
```

Comment:
Is the intention that the user needs to set this to false during SSR, or is it that the hook actually defaults the value to false during SSR?

Reply:
The user needs to set this to false during SSR.

Reply:
It will be force set to false on server, but to avoid hydration mismatch - SSR users has to set it to false